### PR TITLE
Updated fig4 results, new supp. material, increased focus on name origin analysis

### PR DESCRIPTION
**Pull request by nrosed**
In this PR I have 3 main changes:

1. There was a bug in the generation of a table with country affiliations of citations. The DOI to file ID had a shifted index. This affects figure 4 and the final results section.
2. New supplemental data to give some additional context of journalists bias -- specifically looking at the name origin and gender predictions of journalists. This added Supp. tables 5-7 and Supp. Figures 2 and 6. And sections 2 and 3 of the results.
3. I added more emphasis on the East Asian name origin disparity. These changes are focused in the intro and discussion

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ The DOIs were then queried using the _Springer Nature_ API.
 The spider “article_author_crawl.py” scraped all articles designated "Article" or "Letters" from 2005 to 2020.
 We only scraped author names, author positions, and associated affiliations from research articles, which we refer to as _papers_.
 It should be noted that "News" article designations changed over time.
+Additionally, scraping for journalist names was performed months after the initial scraping of the text, and after some aspects of the _Nature_ website changed.
```
Comment (cgreene) [inline content/03.methods.md:19]:
```suggestion
Additionally, scraping for journalist names was performed months after the initial scraping of the text, and some aspects of the _Nature_ website changed.
```

File: content/03.methods.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ The DOIs were then queried using the _Springer Nature_ API.
...
+The website change caused us to lose unique file mappings between the scraped journalist name and the other article information for 137 articles.
```
Comment (cgreene) [inline content/03.methods.md:20]:
```suggestion
The website change caused us to lose unique file mappings between the scraped journalist name and other article metadata for 137 articles.
```

File: content/04.results.md [hunk 3]
```diff
@@ -173,13 +175,21 @@ This reveals a large disparity when considering that people with a predicted Eas
...
 We find a similar pattern when we consider _The Guardian_ articles, except we additionally find a depletion of predicted European name origins (Figure {@fig:suppfig4}c, dark grey and light grey lines).
 
+To further understand the source of Celtic/English over-enrichment and East Asian under-enrichment, we selected a subset of quotes from people that were also cited in the news article; again, this analysis could only be completed on _Nature_ articles.
+We found that the disparity was greatly reduced, more closely matching the analysis on citations alone (Figure {@fig:suppfig_quote_cite}a,b).
+Next, we identified if the journalist name origin had any effect on quote disparities.
+When considering all quotes, we found that journalists with a predicted East Asian name origin had a higher rate of East Asian quoted speakers (Table {@tbl:table5}).
+However, this disparity was reduced when only considering quotes from people cited within the same news article, and reduced further when considering only citations with a US affiliation (Table {@tbl:table6} and {@tbl:table7}).
```
Comment (cgreene) [inline content/04.results.md:182]: I can't quite tell if the disparity that this refers to is the line 181 disparity or the line 178 disparity. Do we definitely mean disparity here?

File: content/05.discussion.md [hunk 4]
```diff
@@ -9,25 +9,42 @@ To examine this last point, we analyzed over 37,000 news articles published in t
...
 We do not believe that the under-quotation is driven by paraphrasing sources, which may occur more frequently with non-native English speakers.
-This is because our findings of under-enrichment of predicted East Asian name origins was recapitulated when we additionally looked at unique names mentioned within news articles.
+We also found that the disparity observed in quotes and mentions was almost eliminated when only considering people that were additionally cited within the same article.
+This suggests that the source of the disparity may lie in the search for additional expert opinions.
+Given that this disparity was more pronounced in _The Guardian_ and its more general target audience, it is possible that the increase in disparity could be driven by fewer citations than in _Nature_ articles.
+However, due to limitations in the API to access _The Guardian_, it remains future work to identify and analyze citations in that publication outlet.
+
+Either way, the clear disparity of predicted East Asian quotes and mentions argues for better source coverage when seeking opinions beyond the academic papers being covered in the article.
```
Comment (cgreene) [inline content/05.discussion.md:32]:
```suggestion
Either way, the clear disparity of predicted East Asian researcher quotes and mentions argues for including a broader set of voices when seeking opinions beyond the academic papers being covered in the article.
```

File: content/05.discussion.md [hunk 5]
```diff
@@ -9,25 +9,42 @@ To examine this last point, we analyzed over 37,000 news articles published in t
...
+One solution could be to have region-specific journalists. 
+This is supported by our analysis of journalists with a predicted East Asian name origin.
```
Comment (cgreene) [inline content/05.discussion.md:34]:
```suggestion
While we were not directly able to examine the regions journalists lived in, this potential strategy is supported by our analysis of journalists with a predicted East Asian name origin.
```

**Review by cgreene — COMMENTED** [review]
I had a few comments and questions - looking good!

File: content/04.results.md (hunk 3 above)
Comment (nrosed) [reply content/04.results.md:182]: good point, I edited it to be "We found that the **_under-enrichment of East Asian name origins_**"

Comment (nrosed) [reply content/04.results.md:182]: changed in https://github.com/greenelab/nature_news_manuscript/pull/11/commits/86381c5ff0d5c571b051bbb12fd326d76303b8ba

File: content/04.results.md [hunk 6]
```diff
@@ -173,13 +175,24 @@ This reveals a large disparity when considering that people with a predicted Eas
 When we again compare _Nature_ articles against last authorship in _Nature_, we observe patterns consistent with the citation analysis with all predicted name origins, except for East Asian and Celtic/English closely matching the predicted name origin rate of last authors in _Nature_ (Figure {@fig:suppfig4}c, dark grey and purple lines).
 We find a similar pattern when we consider _The Guardian_ articles, except we additionally find a depletion of predicted European name origins (Figure {@fig:suppfig4}c, dark grey and light grey lines).
 
+To further understand the source of Celtic/English over-enrichment and East Asian under-enrichment, we selected a subset of quotes from people that were also cited in the news article; again, this analysis could only be completed on _Nature_ articles.
+We found that the under-enrichment of East Asian name origins was greatly reduced, more closely matching the analysis on citations alone (Figure {@fig:suppfig_quote_cite}a,b).
```
Comment (cgreene) [inline content/04.results.md:179]:
```suggestion
We found that the under-enrichment of predicted East Asian name origins was greatly reduced, more closely matching the analysis on citations alone (Figure {@fig:suppfig_quote_cite}a,b).
```

File: content/04.results.md [hunk 7]
```diff
@@ -173,13 +175,24 @@ This reveals a large disparity when considering that people with a predicted Eas
...
+Next, we designed an experiment to test if the predicted journalist name origin had any effect on quote disparities.
```
Comment (cgreene) [inline content/04.results.md:180]:
```suggestion
Next, we designed an experiment to test if predicted journalist name origin had any effect on quote disparities.
```

File: content/04.results.md [hunk 8]
```diff
@@ -173,13 +175,24 @@ This reveals a large disparity when considering that people with a predicted Eas
...
+We found that journalists with a predicted East Asian name origin had a higher rate of East Asian quoted speakers (26.0%) in comparison to journalists with Celtic/English (3.7%) or European (8.3%) predicted name origins (Table {@tbl:table5}).
+To interrogate the source of this difference between journalists with different predicted name origins, we looked at a more specific subsection of quotes.
```
Comment (cgreene) [inline content/04.results.md:182]:
```suggestion
To examine the source of this difference between journalists with different predicted name origins, we looked at a more specific subsection of quotes.
```

File: content/04.results.md [hunk 9]
```diff
@@ -173,13 +175,24 @@ This reveals a large disparity when considering that people with a predicted Eas
...
+We added a new constraint that the quotes must be from a cited last author in the same news article (Table {@tbl:table6}), and that the article must have a US affiliation (Table {@tbl:table7}).
```
Comment (cgreene) [inline content/04.results.md:183]:
```suggestion
We added a new constraint that the quotes must be from a cited last author in the same news article (Table {@tbl:table6}) and that the article must have a US affiliation (Table {@tbl:table7}).
```

File: content/04.results.md [hunk 10]
```diff
@@ -173,13 +175,24 @@ This reveals a large disparity when considering that people with a predicted Eas
...
+We found that differences between journalists with different predicted name origins was nearly eliminated when additionally considering quoted speakers with a US affiliated citation, as evidenced in the East Asian column of Table {@tbl:table7}.
```
Comment (cgreene) [inline content/04.results.md:184]:
```suggestion
We found that differences between journalists with different predicted name origins was nearly eliminated when restricting to quoted speakers with a US affiliated citation, as evidenced in the predicted East Asian column of Table {@tbl:table7}.
```

**Review by cgreene — COMMENTED** [review]
A few suggestions + refined one question

File: content/04.results.md [hunk 11]
```diff
@@ -173,13 +175,27 @@ This reveals a large disparity when considering that people with a predicted Eas
...
+To examine the source of this difference between journalists with different predicted name origins, we looked at a more specific subsection of quotes.
+We added a new constraint that the quotes must be from a cited last author in the same news article (Table {@tbl:table6}) and that the article must have a US affiliation (Table {@tbl:table7}).
+We found that differences between journalists with different predicted name origins was nearly eliminated when restricting to quoted and cited speakers, with marginal improvement with an additional restriction of US affiliated citations, as evidenced in the predicted East Asian column of Table {@tbl:table7}.
+The differences between Table {@tbl:table5} and Tables {@tbl:table6} and {@tbl:table7}, indicate that the predicted name origin of a journalist has some effect on sources gathered outside of directly cited works.
+Since the additional location based constraint of having a US-affilitated citation induces only a marginal change, we are unable to identify if there is a significant regional bias.
```
Comment (cgreene) [inline content/04.results.md:186]:
```suggestion
The additional reduction in differences between the predicted name origin groups and quote rates for US-affiliated people could suggest that, while we don't know the locations of journalists at the time an article is written, regional differences in the presence of journalists may some role in driving the observed disparities.
```

File: content/04.results.md [hunk 12]
```diff
@@ -173,13 +175,27 @@ This reveals a large disparity when considering that people with a predicted Eas
...
+To examine the source of this difference between journalists with different predicted name origins, we looked at a more specific subsection of quotes.
+We added a new constraint that the quotes must be from a cited last author in the same news article (Table {@tbl:table6}) and that the article must have a US affiliation (Table {@tbl:table7}).
+We found that differences between journalists with different predicted name origins was nearly eliminated when restricting to quoted and cited speakers, with marginal improvement with an additional restriction of US affiliated citations, as evidenced in the predicted East Asian column of Table {@tbl:table7}.
+The differences between Table {@tbl:table5} and Tables {@tbl:table6} and {@tbl:table7}, indicate that the predicted name origin of a journalist has some effect on sources gathered outside of directly cited works.
+Since the additional location based constraint of having a US-affilitated citation induces only a marginal change, we are unable to identify if there is a significant regional bias.    
+Furthermore, we do not know the location of the journalists at the time of writing the articles.
```
Comment (cgreene) [inline content/04.results.md:187]:
```suggestion
```

File: content/04.results.md [hunk 13]
```diff
@@ -173,13 +175,27 @@ This reveals a large disparity when considering that people with a predicted Eas
...
+Nevertheless, our findings support the hypothesis that there exist name-origin specific biases of journalists when identifying sources beyond authors directly cited within an articles.
```
Comment (cgreene) [inline content/04.results.md:188]:
```suggestion
```

**Review by cgreene — COMMENTED** [review]
Sorry - I'm still struggling here 😭 . Let me know if things gets things across?

**Review by cgreene — APPROVED** [review]
🚀

**Pull request merged**